src/main/webapp/app/pool-list/pool-list.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+    <h1 class="col-md-12" jhiTranslate="pools.list.title">PCoIP pools</h1>
+</div>
+<div class="row">
+    <div class="col-md-12">
+        <table class="table table-hover table-sm">
+            <thead>
+            <tr class="row">
+                <th class="col-md-3"><p jhiTranslate="pools.list.headers.name">Name</p></th>
```

Comment:
Do we want to implement sorting and filtering data?

if so lets at least make text boxes and flavicons for it.

Reply:
yes we do but again out of scope of this task, it will be done under SNOR-11 Epic